### DISCUSSION TITLE
s/Increase-the-price-of-paid-WeChat-group

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#87] [Announcements]
# s/Increase-the-price-of-paid-WeChat-group

None

https://blog.ops-coffee.cn/s/Increase-the-price-of-paid-WeChat-group

**Comment by ops-coffee** [comment]
想要加群的朋友可以直接点击「运维咖啡吧」微信公众号菜单下的 微信群组 菜单，或微信打开当前原文链接 https://mp.weixin.qq.com/s/N4y-L5vteLKVVcVypSZiWg 点击文章底部的 **喜欢作者** 赞赏即可
